### PR TITLE
TSE REVIEW

### PR DESCRIPTION
**Pull request by carlos-ssh**
**Hi TSE Team I have ready this branch for merge. Please check it out and I'm able for changes and suggestions.
Thanks.**

[Live Demo Link](https://rawcdn.githack.com/carlos-ssh/tnw-page/23e93ca5f80139a758ac379fae0b0253e1017464/index.html)

[Original Page](https://thenextweb.com/)

_Happy Coding!_

**Carlos-SSH**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,11 +1,589 @@
...
+
+                    <article>
+                        <div class="funding-title-div">
+                            <a href="#">
+                                <h2 class="funding-header">Latest funding rounds on Index.co ></h2>
+                            </a>
+                            <ul class="funding-anchors">
+                                <li> <a href="#" style="color: #f42;"> See more</a></li>
+                                <li> <a href="#" style="color: #f42;"> Get updates by email </a></li>
+                                <li> <a href="#" style="color: #f42;"> Add info</a></li>
```
Comment (andresporras3423) [inline index.html]: Don't use inline styles in your HTML.

File: style.css [hunk 2]
```diff
@@ -1,4 +1,795 @@
...
+}
+
+.beige2 {
+  background-color: #fffaf8;
+  height: 45px;
+}
+
+.funding-grid-item4 {
+  display: none;
+}
```
Comment (andresporras3423) [inline style.css]: This selector has the same style definition of the selector on line 133. Group selectors or create a new class.

**Review by andresporras3423 — CHANGES_REQUESTED** [review]
### Status: Changes Required :recycle:
**Hello @AdesojiCodeMaster @carlos-ssh** 
Good job with your new submission for this project :muscle:  
Your page meets most of the requirements :clap: :clap: :clap:  
There are just a few details to improve before you can go ahead:
- [x] Don't open a new pull request for each review.
- [x] Don't repeat yourself, in order to minimize code you can group selectors with the same style definition, or you can create a new class.
- [x]  Don't use inline styles in your HTML
- [ ] I think you need to update the live demo. In your CSS you have the correct breakpoint but your demo uses a different one (with 640px for small screen size).

Please request another review once you are ready! :+1: 

Happy coding :relaxed:

Oscar Andrés Russi Porras

**Comment by carlos-ssh** [comment]
**Hello TSE's we made the next changes:**

-  we don't go to create a new pull request for each review.
- we minimized code, we grouped selectors with the same style definition.
- Deleted inline styles in the HTML file.
- Fixed issues in 640px for small screen sizes.

[Live demo](https://rawcdn.githack.com/carlos-ssh/tnw-page/37b683b03bbfd623ae98bbf04075aed259ad8116/index.html#)

[Original](https://thenextweb.com)

Thanks.

Carlos.

**Review by tirthajyoti-ghosh — APPROVED** [review]
# Approved ✅

## Go ahead and merge it. Your project meets all the requirements.

Happy coding 👨‍💻

**Comment by carlos-ssh** [comment]
Thanks for your review!.

**Pull request merged**